biggraphite/drivers/elasticsearch.py

```diff
@@ -36,11 +36,62 @@
 from biggraphite.drivers import _utils
 from biggraphite.drivers import ttls
 
-UPDATED_ON = prometheus_client.Summary(
-    "bg_elasticsearch_updated_on_latency_seconds", "create latency in seconds"
+READ_ON = prometheus_client.Counter(
+    "bg_elasticsearch_read_on",
+    "Number of updates on the metric field read_on"
 )
-READ_ON = prometheus_client.Summary(
-    "bg_elasticsearch_read_on_latency_seconds", "create latency in seconds"
+UPDATED_ON = prometheus_client.Counter(
+    "bg_elasticsearch_updated_on",
+    "Number of updates on the metric field updated_on"
+)
+
+CLEAN_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_clean_latency_seconds",
+    "clean latency in seconds"
+)
+CREATE_METRIC_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_create_metric_latency_seconds",
+    "create metric latency in seconds"
+)
+DELETE_DIRECTORY_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_delete_directory_latency_seconds",
+    "delete directory latency in seconds"
+)
+DELETE_METRIC_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_delete_metric_latency_seconds",
+    "delete metric latency in seconds"
+)
+DROP_ALL_METRICS_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_drop_all_metrics_latency_seconds",
+    "drop all metrics latency in seconds",
+)
+FLUSH_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_flush_latency_seconds",
+    "flush latency in seconds"
+)
+GET_INDEX_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_get_index_latency_seconds",
+    "get index latency in seconds"
+)
+GLOB_DIRECTORY_NAMES_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_glob_directory_names_latency_seconds",
+    "glob directory names latency in seconds"
+)
+GLOB_METRICS_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_glob_metrics_latency_seconds",
+    "glob metrics latency in seconds"
+)
+METRIC_STATS_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_metric_stats_latency_seconds",
+    "metric stats latency in seconds"
+)
+READ_METRIC_LATENCY = prometheus_client.Summary(
+    "bg_elasticsearch_get_metric_latency_seconds",
```

Comment:
You renamed the variable but not the actual name

Reply:
Sorry, it should be good now.